plots/electricity_bills.py

```diff
@@ -0,0 +1,255 @@
+import os
+import sys
+sys.path.append("../submodules/pypsa-eur")
+import pypsa
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import warnings
+warnings.filterwarnings("ignore")
+
+# get the base working directory
+BASE_PATH = os.path.abspath(os.path.join(__file__ ,"../.."))
+# relative path to folder where to store plots
+PATH_PLOTS = "plots/results/"
+
+def change_path_to_pypsa_eur():
+    # path to pypsa-eur
+    pypsa_path = "submodules/pypsa-eur/"
+    # absolute path to pypsa-eur
+    new_path = os.path.join(BASE_PATH, pypsa_path)
+    # change path to pypsa-eur
+    os.chdir(new_path)
+
+
+def load_networks(lineex, space_resolution, sector_opts, planning):
+    FILE = f"elec_s_{space_resolution}_l{lineex}__{sector_opts}_{planning}_flexible.nc"
+    DIR = "results/flexible/postnetworks"
+    n_flex = pypsa.Network(os.path.join(DIR, FILE))
+
+    FILE = f"elec_s_{space_resolution}_l{lineex}__{sector_opts}_{planning}_retro_tes.nc"
+    DIR = "results/retro_tes/postnetworks"
+    n_retro_tes = pypsa.Network(os.path.join(DIR, FILE))
+
+    FILE = f"elec_s_{space_resolution}_l{lineex}__{sector_opts}_{planning}_flexible-moderate.nc"
+    DIR = "results/flexible-moderate/postnetworks"
+    n_flex_mod = pypsa.Network(os.path.join(DIR, FILE))
+
+    FILE = f"elec_s_{space_resolution}_l{lineex}__{sector_opts}_{planning}_rigid.nc"
+    DIR = "results/rigid/postnetworks"
+    n_rigid = pypsa.Network(os.path.join(DIR, FILE))
+
+    return n_flex, n_flex_mod, n_retro_tes, n_rigid
+
+
+def change_path_to_base():
+    os.chdir(BASE_PATH)
+    # create folder to store images
+    os.makedirs(PATH_PLOTS, exist_ok=True)
+
+
+def get_households():
+    household_data = pd.read_excel("plots/Households.xlsx",skiprows=1,usecols="B:D")
+    households = household_data.set_index("Country")[ "Households (thousands)"]
+    households.index.name=""
+    return households
+
+
+def electricity_bills(network, households):
+    n = network.copy()
+    
+    rh_techs_elec = ['residential rural ground heat pump',
+                     'residential rural resistive heater', 
+                     'residential urban decentral air heat pump',
+                     'residential urban decentral resistive heater']
```

Comment:
missing quite a few...?

```suggestion
    rh_techs_elec = ['residential rural ground heat pump',
                     'residential rural resistive heater', 
                     'residential urban decentral air heat pump',
                     'residential urban decentral resistive heater',
                     'urban central resistive heater',
                     'urban central air heat pump',
                     'residential rural air heat pump']
```

Reply:
Residential rural air heat pump was missing previously, but it was added in recent updated I guess.